src/components/Modal/index.js

```diff
@@ -12,8 +12,13 @@ const Wrapper = ({children}) =>
   wrapWithClass(style.inner, children)
 
 class Modal extends Component {
+
+  static defaultProps = {
```

Comment:
Usually all the default props for the SDK initialisation are inside [index.js](https://github.com/onfido/onfido-sdk-ui/blob/93d5562d21130eb8912d7067bc98eb12135ffe76/src/index.js#L84).

Reply:
should the component level default live at the SDK configuration level? Unless the value is used in multiple other places it seems ok to me to place this default closer to where it will be used.

Reply:
I agree with keeping it closer to where it will be used for a component like this one. This would also ensure that if ever this specific component is brought over to another project the default is brought over also.

Reply:
Good point